apps/blog-app/src/content/2026-08-13-google-merchant-center-ecommerce-surfaces.md

```
@@ -0,0 +1,252 @@
+<figure>
+  <img src="assets/images/blog/2026-08-13-search-console-page-indexing.png" alt="Google Search Console Page indexing report showing 5 indexed pages and 60 not indexed, with two reasons listed" width="100%" height="auto" />
+  <figcaption>The number that explains everything else: 5 indexed, 60 not. No amount of Merchant Center approval competes with a page Google hasn't indexed.</figcaption>
+</figure>
+
+Fifty-nine product pages sat in **"Discovered – currently not indexed"** — Google knew the URLs existed and had decided they weren't worth fetching. One click and three impressions in the performance report is exactly what five indexed pages earns you.
+
+Everything else in Search Console was clean, which is what made this easy to miss: no manual actions, no security issues, HTTPS fine, breadcrumbs valid, sitemap read successfully with all 436 URLs, every product page returning 200 with real server-rendered HTML, a canonical, and complete Product JSON-LD. Every report I'd been checking was green. The one I hadn't opened said the site was effectively invisible.
+
+The cause turned out to be crawl shape, and I found it by curling my own shop page:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)